rotkehlchen/tests/unit/decoders/test_stakedao.py

```diff
@@ -157,3 +178,135 @@ def test_claim_multiple(ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x76d5eb42A854A1cEAfFD99000341d4E4e7a4a70F']])
+def test_deposit(ethereum_inquirer, ethereum_accounts, stakedao_gauges):
+    tx_hex = deserialize_evm_tx_hash('0x0b98f04aeeaa4068b8c8ae0568ed236537c3573b4c3e6fd6b1924741cd5c9ef5')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = ethereum_accounts[0]
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hex)
+    gas_amount, deposit_amount, timestamp = '0.000333982538036425', '29741.066052414178579757', TimestampMS(1742908919000)  # noqa: E501
+    expected_events = [
+        EvmEvent(
```

Comment:
```suggestion
    expected_events = [EvmEvent(
```
styling nitpick: I prefer doing it like this to avoid having it indented more than needed. Also you could even do the assert on this line as well and avoid the extra line for the assert at the end.